Entities/Items/Crate/Crate.as

```diff
@@ -441,7 +423,30 @@ void Unpack(CBlob@ this)
 
 	if (blob !is null && blob.getShape() !is null)
 	{
-		blob.setPosition(this.getPosition() + Vec2f(0, (this.getHeight() - blob.getHeight()) / 2));
+		CMap@ map = getMap();
+		Vec2f pos = this.getPosition() + Vec2f(0, (this.getHeight() - blob.getHeight()) / 2);
+		Vec2f adjusted_pos = pos;
+		if (blob.getName() == "outpost")
+		{
+			// i tried to avoid hardcode but fuck it
+			// prevents outpost from spawning 1 tile above ground
+			pos += Vec2f(0, map.tilesize);
+		}
+
+		if (map.rayCastSolid(this.getPosition(), this.getPosition() + Vec2f(0, 14.0f), adjusted_pos))
+		{
+			adjusted_pos -= Vec2f(0, blob.getHeight() / 2 + 4.0f);
+		}
+		else
+		{
+			if (blob.getName() == "outpost")
+			{
+				//pos -= Vec2f(0, map.tilesize);
```

Comment:
is this suppose to be commented out, if so remove the if empty if statement

Reply:
fixed